VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -1,91 +1,82 @@
-            android:layout_below="@+id/discussion_responses_thread_row_body_text_view">
+            android:orientation="horizontal">
 
             <org.edx.mobile.third_party.iconify.IconView
-                android:id="@+id/discussion_responses_thread_row_pinned_icon_view"
+                android:id="@+id/discussion_responses_thread_closed_icon_view"
                 android:layout_width="@dimen/edx_xxx_small"
                 android:layout_height="@dimen/edx_xxx_small"
-                android:layout_alignParentStart="true"
-                android:layout_alignParentLeft="true"
-                android:layout_centerVertical="true"
                 android:layout_marginEnd="@dimen/discussion_responses_thread_text_view_spacing"
                 android:layout_marginRight="@dimen/discussion_responses_thread_text_view_spacing"
+                android:visibility="gone"
                 app:iconColor="@color/edx_grayscale_neutral_base"
-                app:iconName="fa_thumb_tack"
-                android:visibility="gone"/>
-
-            <RelativeLayout
-                android:id="@+id/discussion_author_relative_layout"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_toEndOf="@+id/discussion_responses_thread_row_pinned_icon_view"
-                android:layout_toRightOf="@+id/discussion_responses_thread_row_pinned_icon_view">
+                app:iconName="fa_lock" />
 
-                <include layout="@layout/discussion_author_layout" />
-
-            </RelativeLayout>
+            <org.edx.mobile.third_party.iconify.IconView
+                android:id="@+id/discussion_responses_thread_row_pinned_icon_view"
+                android:layout_width="@dimen/edx_xxx_small"
+                android:layout_height="@dimen/edx_xxx_small"
+                android:layout_marginEnd="@dimen/discussion_responses_thread_text_view_spacing"
+                android:layout_marginRight="@dimen/discussion_responses_thread_text_view_spacing"
+                android:visibility="gone"
+                app:iconColor="@color/edx_grayscale_neutral_base"
+                app:iconName="fa_thumb_tack" />
 
-            <RelativeLayout
-                android:id="@+id/discussion_number_responses_relative_layout"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_alignParentEnd="true"
-                android:layout_alignParentRight="true"
-                android:gravity="end"
-                android:layout_toRightOf="@+id/discussion_author_relative_layout">
+            <include layout="@layout/discussion_author_layout" />
 
-                <include layout="@layout/number_responses_or_comments_layout" />
+            <View
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_weight="1" />
```

Comment:
You can just add the width, height, and weight dimensions on the previous `include` tag…